### PR TITLE
Update targets UI

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

Update the UI for targets so they are more user friendly. In particular rework how percentages are shown so we no longer show a percentage of a percentage.

medic/medic-webapp#3629
medic/medic-webapp#2522

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: static/css/targets.less [hunk 1]
```diff
@@ -0,0 +1,139 @@
+.targets {
+  display: flex;
```
Comment (SCdF) [inline static/css/targets.less]: 💪

File: static/css/targets.less [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  align-items: stretch;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: 35px 0;
+
+  .target {
+    padding: 10px;
+    width: 350px;
+    min-height: 180px;
+    background-color: #FFFFFF;
```
Comment (SCdF) [inline static/css/targets.less]: Not `white`?

File: static/css/targets.less [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  }
+}
+
+@media (max-width: 400px) {
+  .targets {
+    padding: 0;
+    .target {
+      width: 100%;
+      margin: 0 0 8px 0;
+      border-radius: 0;
```
Comment (SCdF) [inline static/css/targets.less]: I didn't think you needed to do this by default? Or is this overriding something else?

File: static/css/targets.less (hunk 2 above)
Comment (garethbowen) [reply static/css/targets.less]: Personal preference - I never use the reserved words for colours. I don't know why...

File: static/css/targets.less (hunk 3 above)
Comment (garethbowen) [reply static/css/targets.less]: This is overriding the non-mobile styles.

**Pull request merged**